src/ontology/cl-edit.owl

```diff
@@ -24301,7 +24301,7 @@ SubClassOf(obo:CL_1000345 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000198
 
 # Class: obo:CL_1000346 (enterocyte of epithelium proper of large intestine)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm"^^xsd:string) obo:IAO_0000115 obo:CL_1000346 "An enterocyte that is part of the epithelium proper of large intestine."^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm"^^xsd:string) obo:IAO_0000115 obo:CL_1000346 "MERGED DEFINITION: TARGET DEFINITION: A columnar cell of the colon that is part of the epithelium proper of large intestine. SOURCE DEFINITION: An enterocyte that is part of the epithelium proper of large intestine."^^xsd:string)
```

Comment:
@matentzn not sure what this is, I assume this is part of the fix?

Reply:
@shawntanzk I did that edit yesterday (cleaned up def of CL_1000346). Was the edit reverted? Thanks.

Reply:
the def now should contain the whole string "MERGED DEFINITION: TARGET DEFINITION: A columnar cell of the colon that is part of the epithelium proper of large intestine. SOURCE DEFINITION: An enterocyte that is part of the epithelium proper of large intestine." - If possible could you fix this in a separate PR to whichever the correct def is please? Thanks

Reply:
just to be 100% sure, the correct def is "An enterocyte that is part of the epithelium proper of large intestine." right?